homeassistant/components/assist_satellite/models.py

```diff
@@ -0,0 +1,35 @@
+"""Models for assist satellite."""
+
+from dataclasses import dataclass
+from enum import StrEnum
+
+
+@dataclass(frozen=True)
+class SatelliteConfig:
+    """Configuration of satellite."""
+
+    default_pipeline: str | None = None
+    """Pipeline id to use by default (None = preferred)."""
+
+    finished_speaking_seconds: float = 1.0
+    """Seconds of silence before voice command is finished."""
+
+
+class AssistSatelliteState(StrEnum):
+    """Valid states of an Assist satellite entity."""
+
+    WAITING_FOR_WAKE_WORD = "waiting_for_wake_word"
+    """Device is waiting for a wake word."""
+
+    LISTENING_WAKE_WORD = "listening_wake_word"
+    """Device is streaming audio for wake word detection to Home Assistant."""
```

Comment:
We shouldn't differentiate between these two. It's the same state: listening for wake word.